README.md

```diff
@@ -411,65 +440,585 @@ Four actions are available for you to hook onto through the tooltip/popover life
 }}
 ```
 
-## Test Helpers
+## Testing
+
+### Test helpers
 
 This addon exposes testing helpers which can be used inside of the consuming app's acceptance and integration tests. We use a tooltip-centric naming convention but these can also be used to test popovers.
 
-* `assertTooltipVisible(assert)`: asserts if the tooltip is visible. It checks two attributes on the $tooltip: aria-hidden and data-tether-enabled.
-* `assertTooltipNotVisible(assert)`: asserts if the tooltip is not visible. It checks two attributes on the $tooltip: aria-hidden and data-tether-enabled.
-* `assertTooltipRendered(assert)`: asserts if the tooltip has been rendered. When enableLazyRendering is true the tooltip will only be rendered after the user has interacted with the $target element. A tooltip can be rendered but not visible.
-* `assertTooltipNotRendered(assert)`: asserts if the tooltip has not been rendered. When enableLazyRendering is true the tooltip will only be rendered after the user has interacted with the $target element.
-* `assertTooltipSide(assert, { side: 'right' }): asserts that the tooltip is shown on the correct side of the target. Additional options that can be passed are `selector` and `targetSelector`.
-* `assertTooltipSpacing(assert, { side: 'right', spacing: 10 }): asserts that the tooltip is a given distance from the target (on a given side). `side` and `spacing` must be passed. Additional options that can be passed are `selector` and `targetSelector`.
-* `assertContent(assert, { contentString: 'hello' }): asserts that the content of the tooltip matches a given string. 
-* `triggerTooltipTargetEvent($targetElement, eventName)`: triggers an event on the passed element. The event will be triggered within an Ember.run so that the tooltip's asynchronicity is accounted for. eventName can be mouseenter, mouseleave, click, focus, focusin, and blur.
+Publically available test helpers are:
+
+- [assertTooltipContent()](#asserttooltipcontent)
+- [assertTooltipRendered()](#asserttooltiprendered)
+- [assertTooltipNotRendered()](#asserttooltipnotrendered)
+- [assertTooltipVisible()](#asserttooltipvisible)
+- [assertTooltipNotVisible()](#asserttooltipnotvisible)
+- [assertTooltipSide()](#asserttooltipside)
+- [assertTooltipSpacing()](#asserttooltipspacing)
+- [triggerTooltipTargetEvent()](#triggertooltiptargetevent)
+
+All assert helpers require `assert` to be passed as the first param and accept a second, optional param for additional test options. For detailed usage instructions and examples, see the documentation for each test helper below.
```

Comment:
`All assert helpers require `assert` to be passed as the first param and SOME accept a second, optional param for additional test options.`